docs/api/version-history.md

```diff
@@ -15,11 +15,11 @@ keywords: "API, Docker, rcli, REST, documentation"
 
 ## v1.26 API changes
 
-[Docker Engine API v1.26](v1.26/) documentation
+[Docker Engine API v1.26] documentation
 
 ## v1.25 API changes
 
-[Docker Engine API v1.25](v1.25.md) documentation
```

Comment:
I think this will become https://docs.docker.com/engine/api/v1.25/ once 1.13 is released (same for the v1.26 above)